source/Calamari/Kubernetes/ResourceLoggingExtensionMethods.cs

```diff
@@ -11,7 +11,7 @@ public static void LogResources<T>(this ILog log, IEnumerable<T> resourceToLog)
             foreach (var resourceIdentifier in resourceToLog)
             {
                 var hasNamespace = !string.IsNullOrEmpty(resourceIdentifier.Namespace);
-                log.Verbose($" - {resourceIdentifier.Kind}/{resourceIdentifier.Name} {(hasNamespace ? $"in namespace {resourceIdentifier.Namespace}" : "")}");
+                log.Verbose($" - {resourceIdentifier.GroupVersionKind}/{resourceIdentifier.Name} {(hasNamespace ? $"in namespace {resourceIdentifier.Namespace}" : "")}");
```

Comment:
what does this do to the logs, do you get something like `apps/v1/Deployment/Blah`?

Reply:
Yep. The GroupVersionKind `ToString` is used automatically. It ends up with logs like in https://github.com/OctopusDeploy/Calamari/pull/1395/files#diff-0db08797bc823f1629cdd5af8e58f9cff19d259b717e9529f9e9c784e6dacff3